static/versionDetails.json

```diff
@@ -404,6 +404,11 @@
       "flag": "licenseURL",
       "description": "A URL to a licensing agreement text file that will be presented during installation and needs to be accepted for the install to continue.",
       "type": "string"
+    },
+    {
+      "flag": "hostPreflights",
+      "description": "A troubleshoot.sh/v1beta2 spec for adding additional preflight checks prior to a kURL installation. Information on writing preflight checks can be found at https://troubleshoot.sh/docs/preflight/introduction/.",
```

Comment:
Does it have to be this API version? Can't they specify the API version in the spec they provide?

Reply:
kURL will throw an error if it is defined as a different version for the apiVersion of the spec.